### ISSUE TITLE
Include dirs for N-API messed up

### ISSUE THREAD

**Issue by ranisalt** [prebuild/prebuildify#28]
I think this utility is messing up include directories. Building a N-API module yields the following issue:

```
/home/rsa/js/node-argon2/node_modules/node-addon-api/napi.h:4:10: fatal error: node_api.h: No such file or directory
 #include "node_api.h"
          ^~~~~~~~~~~~
```

The same problem does not happen with `node-gyp rebuild`.

Relevant bindings.gyp entries:
```json
"sources": [
  "src/argon2_node.cpp"
],
"include_dirs": ["<!@(node -p \"require('node-addon-api').include\")"],
```

I am unsure how to debug further since prebuildify does not expose node-gyp verbose flag, and I can't see the compiler command line.

**Comment by ranisalt** [comment]
Okay, I managed to spit node-gyp build commands with `export npm_config_loglevel=verbose`, and the faulty command is:

```
g++ (...) -I../argon2/include -I/home/rsa/.node-gyp/5.0.0/include/node (...) -I/home/rsa/js/node-argon2/node_modules/node-addon-api  (...)   -c -o Release/obj.target/argon2/src/argon2_node.o ../src/argon2_node.cpp
```

So the issue seems to be that it is trying to build against Node 5.0, which is not supported by N-API (it requires at least Node 8 for non-experimental support). Furthermore, my package.json engine key should be honored, building only for Node 8.0 and higher.

**Comment by ranisalt** [comment]
So it was my node-gyp installation broken. I had remains of Node 5.0 and node-gyp doesn't care if it was updated ever. Running `rm -rf ~/.node-gyp` fixes it, redownloading Electron 5.0 over old Node 5.0 place.

This will futurely become an issue should Electron versions ever overlap supported Node versions, e.g. Electron reaches 8.0.0 before Node 8.0.0 phases out, as there will be two versions with the same version and different ABIs. Should definitely be looked into.
